lib/components/artist/related_artists/index.js

```diff
@@ -10,6 +10,8 @@ import SerifText from '../../text/serif';
 import ImageView from '../../opaque_image_view';
 import RelatedArtist from './related_artist'
 
+const window = Dimensions.get('window')
```

Comment:
Interesting, so the `width` value still updates after orientation changes?

I actually meant to do it once inside the `columnCount` method, but if this works that’s even better and we can do this in other places too.